conveyal/analysis-ui
# Fix snap to existing stops

## How to test

1. Create an add trip pattern. Edit the geometry so that it passes over stops.
2. Click "Snap to existing stops". Ensure it snaps to stops that it passes over.
